src/components/molecules/Visualizer/Layers/store.ts

```diff
@@ -47,9 +49,11 @@ export class LayerStore {
       },
     );
 
-    this.#root = root;
-    this.#proot = this.#pluginLayer(root);
-    this.#flattenLayers = flattenLayers(root?.children ?? []);
+    this.#root = databaseRootLayer ?? { id: "", children: [] };
+    this.#addedLayersData = [];
+
+    this.#proot = this.#pluginLayer(this.#root);
+    this.#flattenLayers = flattenLayers(this.#root?.children ?? []);
     this.#map = new Map(this.#flattenLayers.map(l => [l.id, l]));
     this.#pmap = new Map(this.#flattenLayers.map(l => [l.id, this.#pluginLayer(l)]));
```

Comment:
```suggestion
    this.setRoot(databaseRootLayer);
```

Can we put together a common process like this?